tsdoc/src/__tests__/DocNodeTransforms.test.ts

```diff
@@ -0,0 +1,27 @@
+import { ParserContext, TSDocParser } from '..';
+import { TestHelpers } from '../parser/__tests__/TestHelpers';
+import { DocParagraph, DocNode, DocNodeKind } from '../nodes';
+import { DocNodeTransforms } from '../transforms/DocNodeTransforms';
+
+test('01 trimSpacesInParagraphNodes()', () => {
+  const buffer: string = [
+    '/**',
+    ' *',
+    ' *    This \t is    the',
+    ' * first   {@mylink}paragraph.',
+    ' * ',
+    ' *         This is the second',
+    ' *    paragraph.',
```

Comment:
If you aren't going to process the second paragraph, may as well remove it from the test.